MekHQ/src/mekhq/campaign/parts/equipment/AmmoBin.java

```diff
@@ -474,7 +491,7 @@ public void updateConditionFromPart() {
 				mounted.setHit(false);
 		        mounted.setDestroyed(false);
 		        mounted.setRepairable(true);
-		        mounted.changeAmmoType((AmmoType) type);
+		        //mounted.changeAmmoType((AmmoType) type);
```

Comment:
Let's get rid of the commented-out code, it's just clutter and we can look it up in source control if we need it later.